src/transcriber.py

```diff
@@ -1,6 +1,6 @@
 import whisper
 
-def transcribe(filename, requestedModel, task, language):
+def transcribe(filename, requestedModel, task, language,email):
```

Comment:
Its not being used?
I am guessing it is only used for logging?

A few linters will try to remove unused args, so someone not knowing the context around why it is here could remove it by mistake

Reply:
Hm. Maybe just logging it would be nice. I'll do that.
´